k8s/charts/netbox-chart/values.yaml

```diff
@@ -329,12 +329,25 @@ podAnnotations: {}
 
 nodeSelector: {}
 
-tolerations: []
+tolerations:
+  - key: CriticalAddonsOnly
+    operator: Equal
+    value: "true"
+    effect: NoSchedule
 
 updateStrategy: {}
   # type: RollingUpdate
 
-affinity: {}
+affinity:
+  nodeAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
+     - preference:
+         matchExpressions:
+         - key: kubernetes.azure.com/mode
+           operator: In
+           values:
+           - system
```

Comment:
i don't think we need this, on the management clusters the 'user' apps are anything non AKS / non flux etc imo

In other words this is a 'user' app on this cluster imo

Reply:
ok 👍